attorch/layers.py

```diff
@@ -755,16 +755,36 @@ def __repr__(self):
             r += '  -> ' + ch.__repr__() + '\n'
         return r
 
-
-class BiasBatchNorm2d(nn.BatchNorm2d):
+class BiasBatchNorm2d(nn.Module):
     def __init__(self, features, **kwargs):
         kwargs['affine'] = False
-        super().__init__(features, **kwargs)
-        self.bias = nn.Parameter(torch.Tensor(features))
+        super().__init__()
+        self.bn = nn.BatchNorm2d(features, **kwargs)
```

Comment:
Shouldn't that have an `affine=False` by default then?

Reply:
`affine=False` would allow the user to override it and also suggests that you can change the behavior. Given this, I prefer to simply fix the value in the init